### PR TITLE
UI: Handle update_properties signal in OBSBasicFilters window

### PR DESCRIPTION
**Pull request by nleseul**
The OBSBasicFilters window did not register a handler for the
"update_properties" signal. Now it does. Addresses [issue #1028](https://obsproject.com/mantis/view.php?id=1028) .

### CODE REVIEW

File: UI/window-basic-filters.cpp [hunk 1]
```diff
@@ -173,6 +174,11 @@ void OBSBasicFilters::UpdatePropertiesView(int row, bool async)
 			(PropertiesReloadCallback)obs_source_properties,
 			(PropertiesUpdateCallback)obs_source_update);
 
+	updatePropertiesSignal.Connect(obs_source_get_signal_handler(filter),
+		"update_properties",
```
Comment (RytoEX) [inline UI/window-basic-filters.cpp]: Style:  Use 2 tabs for indenting line continuations.  See also the next two lines.

File: UI/window-basic-filters.cpp [hunk 2]
```diff
@@ -181,6 +187,12 @@ void OBSBasicFilters::UpdatePropertiesView(int row, bool async)
 	view->show();
 }
 
+void OBSBasicFilters::UpdateProperties(void *data, calldata_t *)
+{
+	QMetaObject::invokeMethod(static_cast<OBSBasicFilters*>(data)->view,
+		"ReloadProperties");
```
Comment (RytoEX) [inline UI/window-basic-filters.cpp]: Style:  Use 2 tabs for indenting line continuations.

**Comment by nleseul** [comment]
Thanks! I've updated the branch. Yes, I was referring to the issue in Mantis.

**Comment by jp9000** [comment]
Thank you for fixing it up, though in the future make sure to squash your commits so that they're all in the same.  I'll squash and merge.  You should see "closed in [commit hash]".

**Pull request closed without merging**